internals/secrethub/generate_test.go

```diff
@@ -46,6 +47,24 @@ func TestGenerateSecretCommand_run(t *testing.T) {
 			err:  nil,
 			out:  "A randomly generated secret has been written to namespace/repo/secret:1.\n",
 		},
+		"copy to clipboard": {
+			cmd: GenerateSecretCommand{
+				generator: randchargeneratorfakes.FakeRandomGenerator{
+					Ret: []byte("random generated secret"),
+					Err: nil,
+				},
+				firstArg:        "namespace/repo/secret",
+				copyToClipboard: true,
+				clipper:         fakeclip.New(),
+			},
+			writeFunc: func(path string, data []byte) (*api.SecretVersion, error) {
+				return &api.SecretVersion{Version: 1}, nil
+			},
+			path: "namespace/repo/secret",
+			data: []byte("random generated secret"),
+			out: "A randomly generated secret has been written to namespace/repo/secret:1.\n" +
+				"The generated value has been copied to the clipboard. It will be cleared after Less than a second.\n",
```

Comment:
Could we also check whether the value has actually been written to the fake clipboard?

Reply:
Done